guides/v2.2/extension-dev-guide/view-models.md

```diff
@@ -0,0 +1,66 @@
+---
+group: php-developer-guide
+subgroup: 99_Module Development
+title: ViewModels
+menu_title: ViewModels
```

Comment:
```suggestion
menu_title: View models
```

Reply:
Since `menu_title` is metadata is no longer necessary I believe this change should go to `title`